src/search/model/SearchUtils.ts

```diff
@@ -83,9 +83,9 @@ export const SEARCH_MAIL_FIELDS: ReadonlyArray<SearchMailField> = [
 		textId: "to_label",
 		field: "to",
 		attributeIds: [
-			typeModels.Mail.associations["toRecipients"].id as number,
-			typeModels.Mail.associations["ccRecipients"].id as number,
-			typeModels.Mail.associations["bccRecipients"].id as number,
+			typeModels.Recipients.associations["toRecipients"].id as number,
```

Comment:
is this the same id as before??

Reply:
tutanota/Mail toRecipients was ID 112
tutanota/Recipients toRecipients is ID 1279

Reply:
we hardcoded the affected IDs now but left the other ones in place because it's good documentation to get them from the models.